Sources/AST/Environment/Information/FunctionInformation.swift

```diff
@@ -24,4 +24,8 @@ public struct FunctionInformation {
   var resultType: RawType {
     return declaration.signature.rawType
   }
+
+  public var requiredParameterIdentifiers: [Identifier] {
```

Comment:
True, I am only fiddling with this because the linking script is broken and apparently modifying this forces it to recompile something (that's my guess).